db/schema/transaction.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const user = require('user.js')
+
+const transactionSchema = new mongoose.Schema({
+    uuid: String,
+    date: String,
+    items: Array,
+    tax: Number,
+    tip: Number,
+    currency: {
+        type:String,
+        default:"USD"

```

Comment:
nit: add space in between + anywhere else

Reply:
will do!